packages/cozy-procedures/src/components/documents/EmptyDocumentHolder.jsx

```diff
@@ -41,7 +41,12 @@ class EmptyDocumentHolder extends Component {
 
       linkDocumentSuccess({ document: createdFile.data, documentId })
     } catch (uploadError) {
-      console.log('errr', uploadError)
+      if (uploadError.status === 409) {
+        Alerter.error(t('documents.upload.conflict_error'))
+      } else {
+        Alerter.error(t('documents.upload.error'))
+        console.log('Upload error : ', uploadError)
```

Comment:
remove `console.log` ?

Reply:
I don't know. Since we don't log anything yet in Sentry... With that we can ask the user to tell us what appears in the console.